pkg/cmd/workflow.go

```diff
@@ -22,130 +21,57 @@ func newWorkflowCommand() *cobra.Command {
 		Args:  cobra.MinimumNArgs(1),
 	}
 
-	cmd.AddCommand(newAddWorkflowCommand())
-	cmd.AddCommand(newReplaceWorkflowCommand())
+	cmd.AddCommand(newSaveWorkflowCommand())
 	cmd.AddCommand(newValidateWorkflowFileCommand())
 	cmd.AddCommand(newDeleteWorkflowCommand())
 	cmd.AddCommand(newRunWorkflowCommand())
 	cmd.AddCommand(newListWorkflowsCommand())
 	cmd.AddCommand(newDownloadWorkflowCommand())
 	cmd.AddCommand(newSecretCommand())
 
+	// Deprecated
+	cmd.AddCommand(newAddWorkflowCommand())
+	cmd.AddCommand(newReplaceWorkflowCommand())
+
 	return cmd
 }
 
-func doAddWorkflow(cmd *cobra.Command, args []string) error {
-	workflowName, err := getWorkflowName(args)
-	if err != nil {
-		return err
-	}
-
-	Dialog.Progress("Creating your workflow...")
-
-	workflow, cwerr := Client.CreateWorkflow(workflowName)
-	if cwerr != nil {
-		return cwerr
-	}
-
-	var fileInfo string
-	if cmd.Flags().Changed("file") {
-		filePath, revisionContent, err := readFile(cmd)
-		if err != nil {
-			return err
-		}
-
-		revision, err := Client.CreateRevision(workflow.Workflow.Name, revisionContent)
-		if err != nil {
-			Dialog.Warnf(`When uploading the file %s, we encountered the following errors:
-
-%s
-
-`,
-				filePath,
-				format.Error(err, cmd),
-			)
-
-			fileInfo = ", but the initial file content contained errors"
-		} else {
-			wr := model.NewWorkflowRevision(workflow.Workflow, revision.Revision)
-			wr.Output(Config)
-
-			fileInfo = fmt.Sprintf(" with file %s", filePath)
-		}
+func newSaveWorkflowCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "save [workflow name]",
+		Short: "Save a Relay workflow",
+		Args:  cobra.MaximumNArgs(3),
+		RunE:  doSaveWorkflow,
 	}
 
-	Dialog.Infof(`Successfully created workflow %v%s.
-
-View more information or update workflow settings at: %v`,
-		workflow.Workflow.Name,
-		fileInfo,
-		format.GuiLink(Config, "/workflows/%v", workflow.Workflow.Name),
-	)
+	cmd.Flags().StringP("file", "f", "", "Path to Relay workflow file")
+	cmd.Flags().BoolP("no-overwrite", "O", false, "Abort instead of overwriting existing revision")
```

Comment:
We should avoid any terminology explicitly referring to "revision" in the documentation.